tap_s3_csv/sync.py

```diff
@@ -193,6 +193,22 @@ def sync_compressed_file(config, s3_path, table_spec, stream):
     return records_streamed
 
 
+def get_source_type_for_updatecol_map(config, source_type_map):
+    column_updates = config['columns_to_update'] if 'columns_to_update' in config else None
+
+    source_type_for_updatecol_map = {}
+    if column_updates and len(column_updates) > 0:
+        updates = list(column_updates.values())[0]
```

Comment:
is the key to column_updates stream? why don't we pass in stream and access column_updates properly instead of converting it to a list?

Reply:
https://varicent.atlassian.net/browse/WP-11496
I will include all above issues and remove `string` in this ticket